src/main/java/core/basesyntax/Application.java

```diff
@@ -1,7 +1,12 @@
 package core.basesyntax;
 
 public class Application {
+    private static final int NumberOfBalls = 3;
```

Comment:
The name of the constant `NumberOfBalls` does not follow the naming conventions for constants in Java. Constants should be named using uppercase letters and underscores to separate words. For example, `NUMBER_OF_BALLS`.

Reply:
fix comment